client/app/domain/redirection/add/domain-redirection-add.controller.js

```diff
@@ -225,7 +225,7 @@ angular.module("controllers").controller(
         ipaddrValid (ip) {
             if (this.Validator.isValidIpv4(ip)) {
```

Comment:
`this.Validator.isValidIpv4(ip)` no longer exists, should use `this.Validator.ip.isValid(ip, { version: 4 });`